### ISSUE TITLE
search buffer refresh takes 3-4 seconds

### ISSUE THREAD

**Issue by arekfu** [pazz/alot#795]
Hi,

Refreshing search buffers consistently takes a long time (3-4 seconds), even when the search result only contains a handful of threads (30). My database contains ~20k messages, but I don't think that is the problem because `notmuch search` is much more snappy. Perhaps it is the UI?

Cheers,
Davide

**Comment by pazz** [comment]
Sorry but we need more input here; I cannot reproduce this.
Do you use any of the fancy theming options that inspect the content of mails for search buffers?
At the very least we need a debug log here.

**Comment by arekfu** [comment]
Yes, I use the `content` option. Removing it makes the UI snappy again.

I am attaching a debug log of a session where I start alot, move around a bit, press `@`, move again and exit (with the `content` theme option).
[alot_anonymized.txt](https://github.com/pazz/alot/files/34080/alot_anonymized.txt)

**Comment by pazz** [comment]
I suspect this is caused by threads with many (or large) mails and a sluggish disk, 
because necessarily, the `content` option causes the interface to operate on the content of mails, and thus has to read them from disk. I am not working on this actively but will leave the issue open for reference and in case someone has a fix. Thanks for reporting this though :)
